.github/workflows/backport-pr.yml

```
@@ -0,0 +1,63 @@
+
+      - name: Check if PR is a backport
+        run: |
+          if [[ "${{ github.event.pull_request.title }}" =~ "backport #" ]]; then
+            echo "BACKPORT=true" >> $GITHUB_ENV
+          else
+            echo "BACKPORT=false" >> $GITHUB_ENV
+          fi
+
+      - name: Extract backport branch and issue number
+        if: env.BACKPORT == 'true'
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2116:style:2:8: Useless echo? Instead of 'cmd $(echo foo)', just use 'cmd foo'

(shellcheck)